src/shared/utils/exportData/getReportCSVObject.ts

```diff
@@ -67,5 +68,6 @@ export const getReportCSVObject = <T>({
     version,
     rawScore: getRawScores(responseValues) || '',
     reviewing_id: reviewedAnswerId,
+    ...(legacyProfileId && { legacy_user_id: legacyProfileId }),
```

Comment:
just to confirm, should not the `legacy_user_id` column be the next with `userId` column ?

Reply:
Valdimir sad in the end